apps/showcase-e2e/src/integration/cypress-helper.spec.ts

```diff
@@ -0,0 +1,35 @@
+describe(`Cypress helpers`, () => {
```

Comment:
shouldn't we actually test the indexedDB content here?

Reply:
Well, this test is for testing the library. Now that it actually works we can move on to update the demo app to manipulate the indexedDb and then we get the values out and assert them. That's a step for next week.

Reply:
I'll spend some time on implementing a form that saves it's content to indexedDb and write some tests to that. That will be the base of a blog post for next week